docs/Configurations.md

```diff
@@ -51,35 +52,33 @@ URL parameters allow customization of the FCA's behavior, enabling testers to ta
+| **mf**               | Connects to MFOS server when set to true; affects how APIs in FCA respond.                                                                                                   | true / false              | Use when testing integration with MFOS server.                                                                      |
+| **systemui**         | Changes the background color to purple and adds a "Launch FCA app" button when set to true, in the context of Ripple devices.                                                | true / false              | Relevant for testing FCA as a base app in Ripple environment.                                                       |
+| **testContext**      | Adds the field 'context' in the mocha report generated when set to true.                                                                                                     | true / false              | Useful for including additional context in test reports.                                                             |
+| **voiceGuidance**    | Enables voice output for navigated menu options in the FCA app when set to true.                                                                                             | true / false              | Enhances accessibility by providing voice guidance for menu navigation.                                              |
+| **reportingId**      | Appends this ID to the report name to form the reporting URL.                                                                                                                | (custom ID)               | Customize the report name for easier identification and tracking.                                                    |
+| **standalone**       | Designed for automated testing scenarios, enabling detailed reports and logs within FCA independently of other apps or frameworks. Set to true for standalone execution.     | true / false              | Essential for tests requiring independence from other apps or frameworks, with detailed reporting and logging needs. |
 
 ## App Types
 
 ### Background
 
-Different app types in FCA denote various categories or classifications of applications that can be tested using the platform. 
+Different app types in FCA denote various SDK's of applications that can be tested using the platform. 
 
 ### Setup
 
-- Define the app types as part of the configuration before initiating tests.
-- This may involve specifying the type of applocation under test, such as media apps, utility apps, etc.
+- Define the app types as part of the configuration before initiating tests (AppType can be sent via the intent message) [Remote Execution](https://github.com/rdkcentral/firebolt-certification-app/blob/docs/Execution.md#remotely).
```

Comment:
Use a relative link when hyperlinking docs in the same repo. i.e. `[Remote Execution](./Execution.md#remotely)`